c/datasetService.c

```diff
@@ -137,6 +140,56 @@ static int serveDatasetContents(HttpService *service, HttpResponse *response){
   return 0;
 }
 
+/* new for ENQ */
+
+static int serveDatasetEnqueue(HttpService *service, HttpResponse *response){
+  zowelog(NULL, LOG_COMP_ID_MVD_SERVER, ZOWE_LOG_DEBUG2, "begin %s\n", __FUNCTION__);
+
+  zowelog(NULL, LOG_COMP_ID_MVD_SERVER, ZOWE_LOG_DEBUG2, "%s\n", __FUNCTION__);
+
+  HttpRequest *request = response->request;
+
+  if (!strcmp(request->method, methodGET)) {
+    char *l1 = stringListPrint(request->parsedFile, 1, 1, "/", 0);
+    char *percentDecoded = cleanURLParamValue(response->slh, l1);
+    char *filenamep1 = stringConcatenate(response->slh, "//'", percentDecoded);
+    char *filename = stringConcatenate(response->slh, filenamep1, "'");
+    zowelog(NULL, LOG_COMP_ID_MVD_SERVER, ZOWE_LOG_DEBUG, "Taking enqueue on: %s\n", filename);
+    fflush(stdout);
+    respondWithEnqueue(response, filename, TRUE);
```

Comment:
Should this be really a GET call? It changes the state.

Reply:
its POST now